sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -214,24 +237,150 @@ export interface DeletedCertificate extends Certificate {
+ * An interface representing options that can be passed to {@link createCertificate}.
  */
-export interface CreateCertificateOptions extends KeyVaultClientCreateCertificateOptionalParams {
+export interface CreateCertificateOptions
+  extends KeyVaultClientCreateCertificateOptionalParams,
+    coreHttp.OperationOptions {
   /**
    * Determines whether the object is enabled.
    */
   enabled?: boolean;
-  /**
-   * Options for this request
-   */
-  requestOptions?: coreHttp.RequestOptionsBase;
 }
 
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link backupCertificate}.
+ */
+export interface BackupCertificateOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link cancelCertificateOperation}.
+ */
+export interface CancelCertificateOperationOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link backupCertificate}.
+ */
+export interface BackupCertificateOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link deleteCertificateOperation}.
+ */
+export interface DeleteCertificateOperationOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link deleteCertificate}.
+ */
+export interface DeleteCertificateOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link deleteContacts}.
+ */
+export interface DeleteContactsOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link importCertificate}.
+ */
+export interface ImportCertificateOptions
+  extends KeyVaultClientImportCertificateOptionalParams,
+    coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link deleteIssuer}.
+ */
+export interface DeleteIssuerOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link setCertificateContacts}.
+ */
+export interface SetCertificateContactsOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link setIssuer}.
+ */
+export interface SetIssuerOptions
+  extends KeyVaultClientSetCertificateIssuerOptionalParams,
+    coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link purgeDeletedCertificate}.
+ */
+export interface PurgeDeletedCertificateOptions extends coreHttp.OperationOptions {}
+
+/**
+ * @interface
+ * An interface representing options that can be passed to {@link updateCertificate}.
+ */
+export interface UpdateCertificateOptions
```

Comment:
`UpdateCertificateOptions` is already defined and exported from another point in this file. This seems like a duplicate definition and export

Reply:
Same goes for `BackupCertificateOptions`. It has been defined and exported twice